### PR TITLE
Contact form Added (html and css)

### PR DESCRIPTION
**Pull request by gersonhiggins**
Add Html form
Add Css styles for Contact form
Linters checkout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,227 +1,293 @@
...
+          </p>
         </div>
+      
+      <form action="https://formspree.io/f/mzbqjbkb" method="post">
 
-        
+        <ul>
+            <li>
+                <label for="name">
+<input type="text" name="name" id="name" required placeholder="   Lorem Ipsum">
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure as required in the project,  name input (can be first name and last name field depending on the template used) has maxlength="30" attribute. so please also add maxlength in the properties of your input :+1:

File: index.html [hunk 2]
```diff
@@ -1,227 +1,293 @@
...
+                </label>
+            </li>
+            <li>
+                <label for="email">
+                    <input type="email" name="email" id="email" required placeholder="   loremipsum@gmail.com">
+                </label>
+            </li>
+            <li>
+                <textarea name="" required placeholder="  Write your message here...."></textarea>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure as required in the project,  Message input has `maxlength="500"` attribute. so please also add maxlength in the properties of your input :+1:

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @gersonhiggins 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by topeogunleye — APPROVED** [review]
### Hi @gersonhiggins, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added max length required attribute ✔️ 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abiy006** [comment]
Hi @gersonhiggins - Nice to see you get approved this section. Well deserve !!

**Comment by gersonhiggins** [comment]
@abiy006 thanks man

**Pull request closed without merging**